packages/notifications/examples/basic.md

```diff
@@ -0,0 +1,212 @@
+### Alert
+
+All usages of `<Close />` must contain an `aria-label` or other assistive technique to have
+discernible text.
+
+```jsx
+const { Field, Label, Radio } = require('@zendeskgarden/react-forms/src');
+
+const initialState = { type: 'success' };
+
+<Grid>
+  <Row>
+    <Col size={12}>
```

Comment:
[nit] not related to the `notifications` part of the PR, but I think the `Col` defaults to `[size="12"]`. You might be able to drop it.